src/wp-includes/class-wp-xmlrpc-server.php

```diff
@@ -335,6 +346,23 @@ public function escape( &$data ) {
 		}
 	}
 
+	/**
+	 * @param IXR_Error|string $error
+	 * @param false $message
+	 */
+	public function error( $error, $message = false ) {
```

Comment:
Is there a reason you've chosen to override rather than modify the parent? I don't have a preference either way so mainly just wanting to understand the reasoning.

Reply:
Thanks, that makes sense.